satpy/tests/reader_tests/test_satpy_cf_nc.py

```diff
@@ -29,6 +29,113 @@
 from satpy.readers.satpy_cf_nc import SatpyCFFileHandler
 
 
+@pytest.fixture(scope="session")
+def _cf_scene_i():
```

Comment:
This function and the other one are very similar. I'm guessing based on what I see that the actual data values don't matter, so can these be one helper function that gets called with the specific array shapes and resolutions provided?

And do they need to be fixtures? Not a huge problem, but right now it seems like they are only used to be saved to NetCDF right away. Maybe the saving should be the function being called from the test (ex. `_create_low_res_test_netcdf()` and `_create_high_res_test_netcdf()`)?

Reply:
OK, I did one function with filename and resolution as arguments to save the netcdf from scene.